clang/test/CheckedCRewriter/arrboth.c

```diff
@@ -1,7 +1,10 @@
-// RUN: cconv-standalone -alltypes %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
-//RUN: cconv-standalone %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_NOALL","CHECK" %s
-// RUN: cconv-standalone %s -- | %clang -c -fcheckedc-extension -x c -o /dev/null -
+// RUN: cconv-standalone -alltypes -addcr %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_ALL","CHECK" %s
+// RUN: cconv-standalone -addcr %s -- | FileCheck -match-full-lines -check-prefixes="CHECK_NOALL","CHECK" %s
+// RUN: cconv-standalone -addcr %s -- | %clang -c -fcheckedc-extension -x c -o /dev/null -
 
+// RUN: cconv-standalone -alltypes -output-postfix=checked %s
+// RUN: cconv-standalone -alltypes %S/arrboth.checked.c -- | diff %S/arrboth.checked.c -
```

Comment:
This is to make sure that doing it twice doesn't change anything? I thought you said during the call that `diff` would not be necessary?

Reply:
Yeah, I was supposed to put this in the original comment. For all of the b_ based tests, asserting that the count was 0 did the trick, but either because of additional whitespace or some other reason, the other autogen'd tests actually did convert and produce output, even though there was no difference between the two. So for ~25% of the tests, I had to assert the count was 0, and the other 75, I had to do this diff. Kind of annoying, but I got really lucky that it split up nice enough so that one script covers one variant and another script covers another.

Reply:
There's probably some underlying reason as to why ~75% of the tests still produce output. If I had to guess, it might be something to do with their formatting and the number of newlines in them. It shouldn't affect the accuracy of the tests or anything, but for my own curiosity